framework/graphics/vulkan_resources_util.cpp

```diff
@@ -351,12 +351,12 @@ bool GetImageTexelSize(VkFormat      format,
 
         if (block_width_ptr != nullptr)
         {
-            *block_width_ptr = format_info.block_extent.width;
+            *block_width_ptr = static_cast<uint16_t>(format_info.block_extent.width);
```

Comment:
if the *block_width_ptr is 16 bit, why isn't block_extent.width?

Reply:
block_extent is of type `VkExtent3D`, so we cannot change it and have to cast.